examples/oft-aptos/evmOAppConfig.ts

```diff
@@ -0,0 +1,121 @@
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities-v3'
+
+import type { OAppNodeConfig, OAppEdgeConfig } from '@layerzerolabs/toolbox-hardhat'
+
+const bscAccountConfig: OAppNodeConfig = {
+    owner: '0xCD5691EA878D262516be278cb46f2fd7fF132083',
+    delegate: '0xCD5691EA878D262516be278cb46f2fd7fF132083',
+}
+
+const ethAccountConfig: OAppNodeConfig = {
+    owner: '0xCD5691EA878D262516be278cb46f2fd7fF132083',
+    delegate: '0xCD5691EA878D262516be278cb46f2fd7fF132083',
+}
+
+const bscOAppConfig: OAppEdgeConfig = {
+    enforcedOptions: [
+        {
+            msgType: 1,
+            optionType: ExecutorOptionType.LZ_RECEIVE,
+            gas: 200000, // gas limit in wei for EndpointV2.lzReceive
+            value: 0, // msg.value in wei for EndpointV2.lzReceive
+        },
+        {
+            msgType: 2,
+            optionType: ExecutorOptionType.LZ_RECEIVE,
+            gas: 200000, // gas limit in wei for EndpointV2.lzCompose
+            value: 0, // msg.value in wei for EndpointV2.lzCompose
+        },
+        {
+            msgType: 1,
```

Comment:
its uncommon for us to include NATIVE_DROP in any enforced options example. I think it would be better to remove it and leave it for advanced users to manually add